### DISCUSSION TITLE
Case convertation

### DISCUSSION THREAD

**Discussion by infectormp** [dbeaver/dbeaver#349] [Q&A] [answered]
Code converted to lowercase if input go in uppercase. Look like code must be still in uppercase.

**Comment by serge-rider** [comment, marked as answer]
By default SQL editor converts keywords case to database default case.
You can change this behavior in Preferences->General->Editors->SQL Editor.
You can disable case auto-conversion or change case on SQL Formatting preference page.

**Comment by infectormp** [comment]
ah, understand now.
